lib/config/presets/internal/replacements.ts

```diff
@@ -698,6 +698,560 @@ const mui: PresetTemplate = {
   title: 'material-ui-to-mui',
 };
 
+const k8sImages = [
```

Comment:
Taken from https://console.cloud.google.com/gcr/images/k8s-artifacts-prod

Reply:
Should we move it to a separate file?

Reply:
maybe a good candidate for a new JSON data file

Reply:
Oh, nice feature, thanks for the hint! I moved it to a date file.